.circleci/config.yml

```diff
@@ -167,6 +170,8 @@ jobs:
     macos:
       xcode: "12.5.1"
     resource_class: large
+    environment:
+        fmt_SOURCE: BUNDLED
```

Comment:
I don't think this is required as you have removed the fmt installation from `setup-macos.sh`, right? It won't be installed.

Reply:
Ah, you are right for the mac build this is true, this would only be needed for the jobs using the docker imnages.